### ISSUE TITLE
Feature Request : Per Arena Kits

### ISSUE THREAD

**Issue by AcesGamingUK** [teozfrank/DuelMe#44]
would it somehow be possible to set a kit per arena allowing i have elemental arenas and id like the issue kits to represent each one, also maybe add ability to add/change kits ingame maybe via gui

**Comment by teozfrank** [comment]
kits are not supported in duelme, you have to rely on another plugin to create them (such as essentials) and use the command to give them. There is no way to create a gui to pick a kit with duelme alone.

**Comment by AcesGamingUK** [comment]
but it alsread has the capability to issue "kits" cant you implument it per arena
![image](https://cloud.githubusercontent.com/assets/16180389/11759269/24ba618c-a06f-11e5-9b79-b61b93f208e6.png)

**Comment by teozfrank** [comment]
Yes but the plugin cannot determine what "kits" if any are there. This is simply a list of commands that are run before the start of a duel.

**Comment by xxroltsxx** [comment]
Then what about a set of commands per arena?

**Comment by teozfrank** [comment]
@xxroltsxx @AcesGamingUK  I think I am just going to add support for Essentials directly so we can choose a kit at the start of a duel.

**Comment by AcesGamingUK** [comment]
that would be great, XD
